src/Symfony/Bridge/Twig/AppVariable.php

```diff
@@ -49,6 +52,17 @@ public function setDebug(bool $debug)
         $this->debug = $debug;
     }
 
+    public function setTwig(Environment $twig)
+    {
+        $this->twig = $twig;
```

Comment:
since the method is never used (after my other comment is accounted for), it should be removed